src/App.tsx

```diff
@@ -1,12 +1,13 @@
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import {Header, Navbar} from "./components";
+import { Header, Navbar } from "./components";
 import LandingPage from "./pages";
 
 import MyGeotabIntroduction from "./pages/myGeotab/introduction";
 import MyGeotabWhatsNew from "./pages/myGeotab/whatsNew";
 import MyGeotabGettingStarted from "./pages/myGeotab/guides/gettingStarted";
 import usingCustomTelematicsDevices from "./pages/myGeotab/guides/usingCustomTelematicsDevices";
 import UsingGoDevices from "./pages/myGeotab/guides/usingGoDevices";
+import MyGeotabDevelopingAddIns from "./pages/myGeotab/guides/developingAddIns";
```

Comment:
A link also needs to be added to the Navbar for this page

Reply:
Updated navbar